ci-operator/step-registry/rosa/cluster/deprovision/rosa-cluster-deprovision-commands.sh

```diff
@@ -43,10 +43,13 @@ while rosa describe cluster -c "${CLUSTER_ID}" ; do
   sleep 60
 done
 
-echo "Deleting operator roles"
-rosa delete operator-roles -c "${CLUSTER_ID}" -y -m auto
-echo "Deleting oidc-provider"
-rosa delete oidc-provider -c "${CLUSTER_ID}" -y -m auto
+if [[ "$STS" == "true" ]]; then
```

Comment:
will this script use for hosted-cp cluster deprovision in future?If yes, operator-roles and oidc-provider need to clean for hosted-cp clusters too. Or we treat hosted-cp cluster as STS cluster too?

Reply:
The deprovision is used for any kind of cluster.